### DISCUSSION TITLE
RustDesk Keyboard functions are not working in latest OS of MAC

### DISCUSSION THREAD

**Discussion by almedina-tech** [rustdesk/rustdesk#9926] [Q&A]
Hi! I got 3 laptops, the 2 laptops have the latest OS while the other one has the outdated OS. Now, when I tried to remote into my windows workstation or any OS workstation I could not type any keys on my keyboard. The RustDesk is updated and freshly installed. I just noticed this, it's happening only in MAC environment with latest OS because of my other laptop with Outdated OS, I haven't experience this issue.

Any solution?

**Comment by rustdesk** [comment]
How about the web client, https://rustdesk.com/web

**Reply by rustdesk** [reply]
You can also try out different input source.

<img width="406" alt="Screenshot 2024-11-15 at 9 50 30 AM" src="https://github.com/user-attachments/assets/8f895ee5-b67d-4f9d-bb32-d57245e7284c">

**Reply by rustdesk** [reply]
possible solutions to make input source 1 work: 

https://github.com/rustdesk/rustdesk/issues/3261#issuecomment-1446360067

and

https://github.com/rustdesk/rustdesk/discussions/7762#discussioncomment-9175161

**Comment by rustdesk** [comment]
@fufesou follow it up for furture query.

**Comment by almedina-tech** [comment]
Thank you! Im gonna try that in the next business day and will let you know which one works
